### PR TITLE
Disable useless travis

### PR DESCRIPTION
**Pull request by jcuquemelle**
## What changes are proposed in this pull request?

(Please fill in changes proposed in this fix)

## How is this patch tested?

(Details)

### Is this a user-facing change?

- [ ] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)

### What component(s) does this PR affect?

- [ ] UI
- [ ] CLI
- [ ] API
- [ ] REST-API
- [ ] Examples
- [ ] Docs
- [ ] Tracking
- [ ] Projects
- [ ] Artifacts
- [ ] Models
- [ ] Scoring
- [ ] Serving
- [ ] R
- [ ] Java
- [ ] Python

### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [ ] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -85,38 +42,6 @@ matrix:
...
-      install:
-        - export PATH="/c/Python37:/c/Python37/Scripts:$PATH"
-        - pip install -r dev-requirements.txt
-        - pip install -r travis/small-requirements.txt
-        - pip install -e .
-      script:
-        - pytest --verbose --ignore=tests/h2o --ignore=tests/keras --ignore=tests/pytorch --ignore=tests/pyfunc --ignore=tests/sagemaker --ignore=tests/sklearn  --ignore=tests/spark --ignore=tests/tensorflow --ignore=tests/keras_autolog --ignore=tests/tensorflow_autolog --ignore tests/azureml --ignore tests/onnx --ignore tests/projects --ignore=tests/xgboost --ignore=tests/lightgbm tests
-    # Run small Python 2.7 tests in the "large" test builder so that we don't spend Travis executor
-    # time running Python 2.7 tests if Python 3 small tests fail.
-    - language: python
```
Comment (jdlesage) [inline .travis.yml]: Large tests are run only on 2.7 (sad) So please don't remove it

File: .travis.yml [hunk 2]
```diff
@@ -10,22 +10,6 @@ matrix:
...
-        - if [[ "$TRAVIS_EVENT_TYPE" == "cron" || ! -z "$CHANGED_FILES" ]]; then
-            INSTALL_LARGE_PYTHON_DEPS=true source ./travis/install-common-deps.sh;
-          fi;
-      script:
-        - if [[ "$TRAVIS_EVENT_TYPE" == "cron" || "$CHANGED_FILES" == *"examples"* ]]; then
-            pytest --verbose tests/examples --large;
-          fi
-        - if [[ "$TRAVIS_EVENT_TYPE" == "cron" || "$CHANGED_FILES" == *"Dockerfile"* ]]; then
-            docker build -t mlflow_test_build . && docker images | grep mlflow_test_build;
-          fi
```
Comment (jdlesage) [inline .travis.yml]: That's the nightly build. Are you they are run on build ?

Comment (jcuquemelle) [reply .travis.yml]: Yes, they are run at each build

File: .travis.yml (hunk 1 above)
Comment (jcuquemelle) [reply .travis.yml]: IRL discussion, large tests are run after this removal

**Comment by jcuquemelle** [comment]
Will only be merged on criteo-master

**Pull request closed without merging**